webapp/src/AppComponents/FileBar.tsx

```diff
@@ -35,6 +35,11 @@ export function FileBar(properties: {
         }}
         onDelete={properties.onDelete}
       />
+      <HelpButton
+        onClick={() => window.open("https://docs.ironcalc.com", "_blank")}
```

Comment:
Are you sure you want:

1. hard code the URL (if the docs change place the link will be broken)
2. Point to the general documentation. Wouldn't the documentation for web app be better? (https://docs.ironcalc.com/web-application/about.html)

Reply:
1. Is there a better way for that?
2. Makes sense